app/Http/routes.php

```diff
@@ -35,7 +35,7 @@
 // Nomination
 Route::post('nomination', ['as' => 'nomination.create', 'uses' => 'NominationController@store']);
 // Admin
-Route::group(['before' => 'role:administrator', 'prefix' => 'admin'], function () {
```

Comment:
ahh yeah right now the only `role` that even exists is admin. good to keep in mind if this comes up in the future though